File: Python/58.最后一个单词的长度.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        n = len(s)
        i = n-1
        cnt = 0
        while i >= 0 and s[i]==' ':
            i = i-1
        if i < 0:
            return 0
        while i >= 0 and s[i]!=' ':
            cnt = cnt + 1
            i = i - 1
        return cnt

File: CPP/58.最后一个单词的长度.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n = s.size();
        int cnt = 0;
        int i = n - 1;
        while(i >=0 && s[i] == ' '){
            i--;
        }
        if(i < 0) return 0;
        while(i >=0 && s[i]!= ' '){
            cnt++;
            i--;
        }
        return cnt;
    }
};